xds/src/test/java/io/grpc/xds/internal/ProtobufJsonConverterTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.protobuf.ListValue;
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+public class ProtobufJsonConverterTest {
+
+  @Test
+  public void testEmptyStruct() {
+    Struct emptyStruct = Struct.newBuilder().build();
+    Map<String, Object> result = ProtobufJsonConverter.convertToJson(emptyStruct);
+    assertTrue(result.isEmpty());
```

Comment:
`assertTrue()` is a bit of a code smell. It works, but if it fails you know nothing about the failure. Truth is nice to use and produces a much better error when it fails. The equivalent here would be `assertThat(result).isEmpty()`, which will print `result` when it fails.

It's not always easy to avoid `assertTrue()`, so it's okay to still be around some. But reduced usage makes tests nicer.